docs/code/introduction/variables_in_code.md

```diff
@@ -0,0 +1,42 @@
+# Variables in Code
+
+## Player Variables in Code
+
+Player variables are only accessible when a game is running. Be prepared that the current player may change in a multiplayer game.
+
+Inside a (game) mode you can access the current player using self.player. Alternatively, you can use self.machine.game.player but be aware that both self.machine.game and self.machine.game.player may be None.
+
+You can use player variables like this:
+
+```
+player = self.machine.game.player    # do not persist the player because it may change
+                                     # alternatively use self.player in modes
+
+if not player:
+   return    # do something reasonable here but do not crash in the next line
+
+# read player variable
+print(player["my_variable"])
```

Comment:
I'd discourage the use of `print()` in the example code, I think it'd be just as clear using a variable assignment as the example.

Reply:
The print came from the original docs, I changed it to log but left a note that print works as well. Might be an easier entry for non-Python developers - you never know...